onnx/external_data_helper.py

```diff
@@ -93,31 +97,39 @@ def set_external_data(tensor,  # type: TensorProto
             entry.value = str(v)
 
 
-def convert_model_to_external_data(model, all_tensors_to_one_file=True, location=None):
+def convert_model_to_external_data(model, all_tensors_to_one_file=True, location=None, size_threshold=1024):
     # type: (ModelProto, bool, Optional[Text]) -> None
     """
-    call to set all tensors as external data. save_model saves all the tensors data as external data after calling this function.
+    Call to set all tensors with raw data as external data. This call should preceed 'save_model'.
+    'save_model' saves all the tensors data as external data after calling this function.
```

Comment:
Could we just call save_model as the last step of this function and rename it "save_model_with_external_data" or something like that? What is the value in splitting it into two different calls?

Reply:
I don't know why this was split in 2 calls... I am planning to add another method which will do all this in a single call. For now let's keep this method because there are partners who have a dependency on this and I don't want to change this before prior notification.